--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -209,14 +209,14 @@ include(GNUInstallDirs)
 set (LIB_INSTALL_DIR "${CMAKE_INSTALL_LIBDIR}" CACHE PATH "Installation path for libraries")
 
 # Make a version file containing the current version from git.
-include (GetGitRevisionDescription)
-git_describe (VERSION "--tags")
+#include (GetGitRevisionDescription)
+#git_describe (VERSION "--tags")
 # If git_describe fails, use a dirty version
-if (${VERSION} MATCHES -NOTFOUND)
-  set (VERSION "v1.6.0")
-  message (WARNING "Failed to retrieve UPM version with 'git describe' (using "
-      "${VERSION}). Check that git is installed and this is a valid git repo.")
-endif ()
+#if (${VERSION} MATCHES -NOTFOUND)
+set (VERSION "v1.6.0")
+#  message (WARNING "Failed to retrieve UPM version with 'git describe' (using "
+#      "${VERSION}). Check that git is installed and this is a valid git repo.")
+#endif ()
 
 message (STATUS "UPM Version ${VERSION}")
 
